src/test/test_bitcoin.cpp

```diff
@@ -69,11 +69,11 @@ TestingSetup::TestingSetup(const std::string& chainName) : BasicTestingSetup(cha
         pblocktree = new CBlockTreeDB(1 << 20, true);
         pcoinsdbview = new CCoinsViewDB(1 << 23, true);
         pcoinsTip = new CCoinsViewCache(pcoinsdbview);
-        InitBlockIndex(chainparams);
+        BOOST_REQUIRE(InitBlockIndex(chainparams));
         {
             CValidationState state;
             bool ok = ActivateBestChain(state, chainparams);
-            BOOST_CHECK(ok);
+            BOOST_REQUIRE(ok);
```

Comment:
Meh, in the tests it's pretty normal to check something with side effects. This is only an "assertion" in the abstract sense, not the C/C++ sense that it will potentially be disabled (new: tests in release mode, with disabled tests!).

Anyhow let's not nitpick this two-line change to death.